test/system/kubernetes/fluentBitSetup.sh

```diff
@@ -0,0 +1,665 @@
+#!/usr/bin/env bash
+
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
One last thing: this header is obsolete, please change it to the new one updated few days ago.